src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+import { Response } from '../../../api-objects/GenericResponse';
+import { AdminPatientResponse, ResetPasswordResponse, UserListItem } from '../../../api-objects/AdminPatient';
+import { PatientResponse } from '../../../api-objects/patientResponse';
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createPatient(patientId : string, deceased : boolean, patientNotes : string) : Observable<Response<PatientResponse>> {
+    return this.serverApiService.post('patient', {'patientId': patientId, 'deceased': deceased, 'patientNotes': patientNotes});
```

Comment:
{'patientId': patientId, 'deceased': deceased, 'patientNotes': patientNotes}
Could be rewritten as:
{patientId, deceased, patientNotes}
https://eslint.org/docs/rules/object-shorthand